components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -320,24 +320,7 @@ def __init__(this, name, help, wait=False):
             "--finish", action="store_true",
             help="Re-enables the background indexer after for indexing")
 
-        ports = Action(
-            "ports",
-            """Allows modifying the ports from a standard OMERO install (deprecated)
-
-To have multiple OMERO servers running on the same machine several ports
-must be modified from their defaults. Changing the ports on a running server
-will be prevented, use --skipcheck to override this.
-
-Examples:
-
-  # Set ports to registry:14061, tcp:14063, ssl:14064, web:14080
-  %(prog)s --prefix=1
-  # Set ports back to defaults: 4061, 4063, 4064, 4080
-  %(prog)s --prefix=1 --revert
-  # Set ports to: 4444, 5555, 6666, 7777
-  %(prog)s --registry=4444 --tcp=5555 --ssl=6666 --webserver=7777
-
-""").parser
+        ports = Action("ports", SUPPRESS).parser
```

Comment:
This came from [this discussion](https://github.com/openmicroscopy/openmicroscopy/pull/4093#commitcomment-13534609). The main point is that this command is now a no-op and adding `(deprecated)` to the help does not convey exactly that messafe.